packages/db/src/schema/deployment.ts

```diff
@@ -74,3 +75,25 @@ export const deploymentDependency = pgTable(
   },
   (t) => ({ uniq: uniqueIndex().on(t.dependsOnId, t.deploymentId) }),
 );
+
+export const deploymentLifecycleHook = pgTable("deployment_lifecycle_hook", {
+  id: uuid("id").primaryKey().defaultRandom(),
+  deploymentId: uuid("deployment_id")
+    .notNull()
+    .references(() => deployment.id, {
+      onDelete: "cascade",
+    }),
+  runbookId: uuid("runbook_id")
+    .notNull()
+    .references(() => runbook.id, {
+      onDelete: "cascade",
+    }),
+});
```

Comment:
that implies it does all types of hooks
not just runbook hooks

what would the tables be called for webhooks?

Reply:
should be also include the type of hook this is? target delete, deployment create, deployment delete, env delete? deployment update?

Reply:
we can add this stuff in once we use more than one type of hook